aten/src/ATen/native/native_functions.yaml

```diff
@@ -5410,6 +5410,15 @@
     CompositeExplicitAutograd: slice
   tags: core
 
+# TODO: do I need slice_strict_inverse?
```

Comment:
IIRC (cc @jbschlosser), the corresponding `slice_inverse` basically exists today just to handle the situation where you do something like this:
```
x = TensorSubclass(...)
y = aten.slice(x, ...)
# y becomes an input to a compiled region, need to fakeify
# This runs the "play view inverses to generate fake base, then play view chain to regenerate view with proper metadata" logic
y_fake = FakeTensorMode.from_tensor(y)  
```

So one argument: if we think `slice_strict` will basically never be called in eager mode (and therefore never show up in a view chain we need to fakeify, we could probably pretty reasonably not bother adding it and raise an error when the inverse is supposed to be called during fakeification?

If we want to add it for consistency though (and hide it from python), that also seems relatively harmless